### PR TITLE
With custom button

### PR DESCRIPTION
**Pull request by mbappai**
Added a way to use a custom component to trigger paystack inline. Let me know what you think. Thanks!

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default {
   ],
   watch: {
     clearScreen: false
-  }
+  },
```
Comment (iykekings) [inline rollup.config.js]: can you revert this?

File: src/App.svelte [hunk 2]
```diff
@@ -1,9 +1,12 @@
 <script>
   import Paystack from "./Paystack.svelte";
+  import Button from "./customButton.svelte"
+
   let config = {
     key: "pk_test_a869b0564dc0763418d5cfb3b3b7a2590b1312c6",
     email: "iykekings36@gmail.com",
     amount: 10000,
+    withSlot:false,
```
Comment (iykekings) [inline src/App.svelte]:
```suggestion
    withSlot: false,
```

File: src/Paystack.svelte [hunk 3]
```diff
@@ -1,8 +1,14 @@
 <script>
+  // IMPORTS
   import { onMount } from "svelte";
+
+  //PROPS
+  export let config = {};
+  // export let customButton='';
+
+  //STATES
```
Comment (iykekings) [inline src/Paystack.svelte]: Remove comments:
In svelte, exported variables are props by default and non-exported are states. Also remove the one on the imports

File: src/customButton.svelte [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<script>
+    export let action=()=>null;
+</script>
+
+<button class="cta" on:click={action} >
+    Custom button &#129409
```
Comment (iykekings) [inline src/customButton.svelte:6]: why is this the label of the custom button?

File: src/customButton.svelte [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+</button>
+
+<style>
+    .cta{
+        border-radius: 50px;
+       white-space: nowrap;
+       text-decoration: none;
+       border:0;
+       /* padding: 1rem; */
+       /* margin: 0.25rem; */
```
Comment (iykekings) [inline src/customButton.svelte:16]: remove comments

**Review by iykekings — CHANGES_REQUESTED** [review]
@mujeex can you attend to the review comments?

File: rollup.config.js (hunk 1 above)
Comment (mbappai) [reply rollup.config.js]: What exactly should I revert here? Wasn't it already there before? Haha sorry can't really remember

File: src/Paystack.svelte (hunk 3 above)
Comment (mbappai) [reply src/Paystack.svelte]: Oh my bad, I just did that for sake of clarity sometimes. I easily get things mixed up.

File: src/customButton.svelte (hunk 4 above)
Comment (mbappai) [reply src/customButton.svelte:6]: Sorry I was quite a bit in a rush back there! Do you have a better label for it?

**Review by mbappai — COMMENTED** [review]
Let me know if there is anything I can do.

**Review by iykekings — APPROVED** [review]
Thanks for this. Totally forgot about it

**Pull request merged**